package.json

```diff
@@ -1,29 +1,35 @@
 {
   "name": "@dbos-inc/eslint-plugin",
-  "version": "0.0.6",
+  "version": "0.0.7",
```

Comment:
This is a major change so I'd suggest a new major version `1.0.0`.

Reply:
Got it